tests/integration/solidity_contracts/EFTests/test_sstore.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import pytest
+from starkware.starknet.testing.contract import StarknetContract
+
+from tests.utils.helpers import (
+    hex_string_to_bytes_array,
+    private_key_from_hex,
+)
+from tests.utils.uint256 import hex_string_to_uint256
+
+logger = logging.getLogger()
+
+
+@pytest.mark.asyncio
+@pytest.mark.EF_TEST
+@pytest.mark.SSTORE
+class TestSSTORE:
+    @pytest.mark.skip("TODO: Fix address collision with existing contract on CREATE")
```

Comment:
btw, this will be fixed when putting all the storages in kakarot main contract @enitrat